server/events/vcs/github_client.go

```diff
@@ -296,12 +297,62 @@ func (g *GithubClient) PullIsApproved(repo models.Repo, pull models.PullRequest)
 	return approvalStatus, nil
 }
 
+// GetCombinedStatusMinusApply checks Statuses for PR, excluding atlantis apply. Returns true if all other statuses are not in failure.
+func (g *GithubClient) GetCombinedStatusMinusApply(repo models.Repo, pull *github.PullRequest) (bool, error) {
+	status, _, err := g.client.Repositories.GetCombinedStatus(g.ctx, repo.Owner, repo.Name, *pull.Head.Ref, nil)
+	if err != nil {
+		return false, errors.Wrap(err, "getting combined status")
+	}
+
+	//compile for performance reasons
+	matcher, _ := regexp.Compile("atlantis/apply")
```

Comment:
We have to pass `vcsstatusname` down here since it's actually configurable. See the [counterpart of GitLab](https://github.com/runatlantis/atlantis/blob/master/server/events/vcs/gitlab_client.go#L216) as an example.

Reply:
Forgot about that - will fix.